docs/root/start/docker.rst

```diff
@@ -50,13 +53,29 @@ Build the Docker image using:
    $ docker build -t envoy:v1 .
 
 Assuming Envoy is configured to listen on ports ``9901`` and ``10000``, you can now start it
-with:
+in Docker with:
 
 .. code-block:: console
 
    $ docker run -d --name envoy -p 9901:9901 -p 10000:10000 envoy:v1
 
-Permissions for running Docker Envoy containers as a non-root user
+or in Podman (unsupported) with:
+
+.. code-block:: console
+
+   $ podman run -d --name envoy -p 9901:9901 -p 10000:10000 envoy:v1
+
+Root filesystem permissions for running Envoy in containers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Envoy container image must be run with the container's root filesystem mounted read-write.
```

Comment:
i dont think this is correct - i didnt try it, but i dont see any reason why it cant be mounted read-only - unless logging, caching, or some other use of the fs is configured

Reply:
Admittedly, it's a conservative take. I don't currently have the latitude to comprehensively test RO myself. And I'm not a person that can assert on behalf of the Envoy project that RO is supported. Unless someone has better info ;)

Reply:
i think its wrong - for example testing with:

```console
$ docker run -p 10000:10000 -v $PWD/configs/:/configs --read-only envoyproxy/envoy:distroless-v1.30-latest -c /configs/envoy-demo.yaml
```

works as expected

Reply:
This is exactly what I want -- Acknowledgement, blessing, instructions, and official support from the Envoy project on running Envoy containers read-only. Lemme update this PR.